frontend/src/app/collaboration/editor/editor.component.css

```diff
@@ -0,0 +1,8 @@
+::ng-deep .p-scrollpanel-content {
+    padding: 0 18px 0 0;
+    height: 100%;
+}
+
+::ng-deep .p-component {
+    width: 100%;
+}
```

Comment:
```suggestion
:host ::ng-deep .p-scrollpanel-content {
    padding: 0 18px 0 0;
    height: 100%;
}

:host ::ng-deep .p-component {
    width: 100%;
}
```

Not sure what this does, but it seems to be spilling over into the width of the navigation bar.
![image](https://github.com/user-attachments/assets/664526db-4767-4f12-8c02-355358bbafda)